doc/getting-started.md

```diff
@@ -390,14 +395,62 @@ This will configure the authenticator.
 
 #### Rundeck Setup
 ##### Setup Application
+Rundeck needs an account to access Bitbucket later. We will create an ssh keypair for this and add this later to the Bitbucket `cd_user` account.
+
+Open the shell and generate a ssh key. On cygwin enter the following command:
+```
+ssh-keygen -f cd_user -t rsa -C "CD User"
+```
+This saves the public and private key in a file `cd_user.pub` and `cd_user`.
+
+
+Create a file called `/home/vagrant/rundeck_vars.yml` that customizes some of the rundeck configuration, e.g. the ssh key.
+
+This is a yaml file, looking structurally like this Example
+
+```yaml
+rundeck_bitbucket_host_external: 192.168.56.31
+rundeck_bitbucket_host_internal: localhost
+rundeck_bitbucket_port: 7999
+rundeck_cduser_name: cd_user
+rundeck_cduser_private_key: |
+  -----BEGIN RSA PRIVATE KEY-----
+  MIIJKgIBAAKCAgEA9byVUZKe0dB0gkFL5g4Zcxb3AUNPvtD2tpkejyaLoF/XnQj+
+  qn+UX9WZSn0YyTQH+cmNF1SFuMmq/eSZpdAL7JSRY2bAw9RLo3dPpabO2N3Teib1
+  HSvCnPncNQZa/tPUaWSddX0BTWEpS1fAl4NFfUmN02k+cEHIErv2OcbhMnq675aO
+  p4rU3NHN01kymhUCLz5cUCAj4CyEhxv3Fe7zSeKGuSceaD2Yq1vEnp8WmYnqdiFf
+  ....
+  0rMrGoSgTuttxQ+oU2a+2pRQD+vFXg6BpXMJNXeXyPuSIVfqfSFTqUdshZC8d76Q
+  8IwfUR/GtEjTO4l9nDr0eqb4LixvpREVVvMOH+Ea/a8yATejH9xR7xNHAA0AQqZ+
+  t1pNCqijBNTk2oUYNu9t9m16zF3Ly+ZIikBm0D67ke5yC5ziSPa1Xs6E70ens04H
+  RwP9We5Y453L2st43FlQXVAyXd4OacJcUqvYqQpd7c7u1syhpRzG5ALYcfoNJA==
+  -----END RSA PRIVATE KEY-----
+```
+
+You have to replace the private key with the key you created earlier and change
+other variables according to your environment.
+
+Now execute the playbook:
+
 ```
-ansible-playbook -v -i inventories/dev playbooks/rundeck.yml --ask-vault
+ansible-playbook -v -i inventories/dev playbooks/rundeck.yml -e "@/home/vagrant/rundeck_vars.yml" --ask-vault
 ```
+
+You can change `host` and `cduser` according to your environment.
+<!-- TODO
+This is superfluous if we mirror the repos first to our vagrant / local bitbucket server.
+-->
 After the playbook has been finished Rundeck is accessible via http://192.168.56.31:4440/rundeck
 
+
+### Mirror OpenDevStack Repos 
+<!-- TODO -->
+
+We will create an opendevstack project in our Bitbucket instance and mirror the opendevstack repositories from github to this local project.
```

Comment:
We should make it more clear what the expectancy here is. Do we want the user to mirror all repos, or just selected ones?

The more I think about this question, I think it makes sense for everyone to clone all repos, not just the quickstarters. The reason is that this provides the user with a lot of control over updates (and in fact is how we describe the update procedure) plus it is the easiest pathway to contribution, which we want to encourage.